### PR TITLE
fix(Header): Add mobile menu space

### PR DESCRIPTION
**Pull request by c2pig**
In mobile view : 
- Add hamburger bar toggle
- Add space on top of menu when it trigger from top hamburger bar toggle

New:
Screen Shot 2019-09-04 at 2 51 23 PM

Old:
Screen Shot 2019-09-04 at 2 53 46 PM

### CODE REVIEW

File: react/Header/Header.less [hunk 1]
```diff
@@ -101,3 +102,17 @@
   width: 100%;
   height: 100%;
 }
+
+.svg {
+  fill: 0x0c4b85;
```
Comment (ChanJit) [inline react/Header/Header.less]: the color code is very weird.....

File: react/Header/components/Menu/Menu.js [hunk 2]
```diff
@@ -79,10 +79,12 @@ export default class Menu extends Component {
...
   }
   render() {
-    const { menuMessage, headerMessage, shouldShowMenu, links, locales, brandStyles, employerSite, authenticationStatus, userName, linkRenderer, loginAvailable } = this.props;
+    const { menuMessage, headerMessage, shouldShowMenu, shouldShowMenuWithSpace, links, locales, brandStyles, employerSite, authenticationStatus, userName, linkRenderer, loginAvailable } = this.props;
     const selectedLocale = locales[0];
+    const openMenuWithSpace = shouldShowMenu && shouldShowMenuWithSpace;
+    const openMenu = shouldShowMenu && !shouldShowMenuWithSpace;
     return (
-      <div className={classnames(styles.root, { [styles.showMenu]: shouldShowMenu })}>
+      <div className={classnames(styles.root, { [styles.showMenu]: openMenu, [styles.showMenuSpace]: openMenuWithSpace })}>
```
Comment (ChanJit) [inline react/Header/components/Menu/Menu.js]: unit test?

File: react/Header/Header.js [hunk 3]
```diff
@@ -105,12 +106,15 @@ export default class Header extends Component {
...
     } = this.props;
+    const { showTray } = actionTrayProps;
     const menuOpen = this.state.menuOpen;
+    const shouldShowMenuWithSpace = !showTray;
 
     return (
       <header className={styles.root}>
         <PageBlock>
           <div className={classnames({ [styles.primaryNavNoLogin]: !loginAvailable }, styles.primaryNav)}>
+            {!showTray && <div/> }
```
Comment (ChanJit) [inline react/Header/Header.js]: so is a `space`? 😂 why showing the `<div/>`?

File: react/Header/Header.less [hunk 4]
```diff
@@ -101,3 +102,17 @@
...
+  width: @grid-base * 6;
```
Comment (ZhenYongSeekAsia) [inline react/Header/Header.less]: i think the icon's size is control by props `size`

File: react/Header/Header.js (hunk 3 above)
Comment (c2pig) [reply react/Header/Header.js]: what would you suggest?

File: react/Header/Header.less (hunk 1 above)
Comment (c2pig) [reply react/Header/Header.less]: change it to rgb :D

File: react/Header/Header.js (hunk 3 above)
Comment (c2pig) [reply react/Header/Header.js]: it is for align logo in centre, by using space-between.

File: react/Header/Header.less (hunk 4 above)
Comment (c2pig) [reply react/Header/Header.less]: thanks for spot that.  Did not aware of that have been added.  Will remove it now

**Pull request merged**